guides/common/modules/proc_associating-partition-tables-with-disk-encryption.adoc

```diff
@@ -0,0 +1,28 @@
+[id="associating-partition-tables-with-disk-encryption_{context}"]
+= Associating partition tables with disk encryption
+
+{Project} contains partition tables that encrypt the disk of your host using Linux Unified Key Setup (LUKS) during host provisioning.
+Optionally, you can also bind the disk to a Tang server through Clevis for decryption during boot.
+You must associate the partition table with your operating system entry.
+Then, you assign the partition table to your host group or select it manually during provisioning.
+
+Encrypted disks on hosts protect data at rest.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *Provisioning Setup* > *Operating Systems*.
+. Select your {client-os} entry.
+. On the *Partition Table* tab, associate the `{client-provisioning-template-type} default encrypted` to your operating system entry.
+For more information, see xref:creating-partition-tables_{context}[].
+. Create a host group that uses the `{client-provisioning-template-type} default encrypted` partition table.
+For more information, see {ManagingHostsDocURL}Creating_a_Host_Group_managing-hosts[Creating a host group] in _{ManagingHostsDocTitle}_.
++
+There are two options to decrypt the disk of your hosts during boot time:
++
+* You can add the host parameter `disk_enc_passphrase` as type *string* and your cleartext passphrase of the LUKS container as the value.
+* You can add the host parameter `disk_enc_tang_servers` as type *array* and your list of Tang servers (example: `["1.2.3.4"]` or `["{server-example-com}", "5.6.7.8"]`) to use Clevis/Tang.
++
+If you set `disk_enc_tang_servers`, you do not have to set `disk_enc_passphrase` because the passphrase slot is removed from the LUKS container after provisioning.
+
+.Additional resources
+* https://github.com/latchset/clevis
```

Comment:
If this section is used by Satellite, I recommend that you comment them out. We are not supposed to link to GitHub repos without explicit permission because we cannot guarantee third party content.

Reply:
Hidden for satellite.